data/transport missions.txt

```diff
@@ -1867,6 +1872,297 @@ mission "Terraforming 5"
+			`	The heat has also formed a huge circle of melted snow around the volcano, revealing the green color of trees and the dull grays and browns of rock.`
+			choice
+				`	"What do we do now?"`
+					goto what
+				`	"It looks beautiful from up here."`
+			
+			`	"Yes it does, Captain," the Republic representative says. "Yes it does."`
+				goto end
+			
+			label what
+			`	"Now we wait," Amy says to you. "We should be able to tell how this will affect Tundra's climate in a year or two."
+			
+			label end
+			`	You steer your ship back to the spaceport. Amy pays you <payment> after you land. "Thanks for your help, Captain. We're going to stay here and monitor Tundra's climate as it shifts, so there's nothing more that we need you for."`
+
+
+
+event "terraforming Tundra"
+	planet "Tundra"
+		description `Millions of years ago, this was a warm world, perhaps even tropical. But some cataclysmic event, perhaps a meteor strike or a massive volcanic eruption, altered the planet's atmosphere enough to turn it into the nearly lifeless, frozen planet it is today. Recent experiments in terraforming the planet have attempted to remedy the climate by causing a supervolcano to prematurely erupt.`
+		description `	The first settlers came here to drill for the oil trapped deep under the surface, a relic of Tundra's former, more lively days. Instead of refining the oil into plastic, which sells relatively cheap in this region, they have developed an industry in synthetic fabrics and clothing.`
+
+
+
+event "terraforming timer 2"
+
+mission "Terraforming Follow-up"
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
+		near "Sol" 100
+		attributes spaceport
+	to offer
+		has "event: terraforming timer 2"
+	on offer
+		event "terraforming follow-up"
+		conversation
+			`When you land, you receive a message from Amy, the girl who you helped to terraform Rand and Tundra.`
+			`	"I hope everything is going well, Captain. I just wanted to give you an update on what has occurred in the past year.`
+			`	"Tundra's climate has drastically improved over the past year. Areas around the equator are starting to become completely snow free, and I was amazed when I was able to walk outside without a heavy coat on. The Republic has placed a halt on any more experiments, as they wish to see how the situations on Tundra and Rand turn out.`
+			`	"Speaking of Rand, last month I got to catch up with Eric and Alaric when I visited the planet. They've told me that Rand is becoming a far better place to live on now. It's even becoming a popular tourist destination. They wanted me to relay their gratitude to you for making their dream come true.`
+			`	"That's all I have to say. Thanks again for all the help you've provided to me. I don't know where I would be without it."`
```

Comment:
`all the help you've provided to me` sounds weird. Try removing `to me` and replacing `provided` with a different verb.
Or we can replace that whole bit with `all your help`.